torchtitan/models/llama/model.py

```diff
@@ -132,7 +132,6 @@ class Attention(nn.Module):
     Attributes:
         n_kv_heads (int): Number of key and value heads.
         n_heads (int): Number of query heads.
-        n_local_kv_heads (int): Number of local key and value heads.
```

Comment:
not an attribute (only one occurrence of `n_local_kv_heads` if you search in this file)